tdrs-backend/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       - "9000:9000"
 
   localstack:
-    image: localstack/localstack:0.12.9
+    image: localstack/localstack:0.13.3
```

Comment:
I was having issues in my local environment with v12 so I have upgraded to latest without issues.